client/my-sites/backup/status/style.scss

```diff
@@ -1,18 +1,3 @@
-.status__card-list {
-	margin: 0;
-	padding: 0;
-
-	list-style-type: none;
-
-	> li {
-		margin: 44px 0;
-
-		&:first-child {
-			margin-top: 0;
-		}
-	}
-}
-
-.is_jetpackcom .status__card-list {
+.is_jetpackcom {
 	margin-top: 32px;
 }
```

Comment:
Why to leave this here?

`.is_jetpackcom .status__card-list` selector will select `.status__card-list` that is a decedent of `.is_jetpackcom`, but `.is_jetpackcom` is different than it was before.

I think we need to remove this file and remove its import from `client/my-sites/backup/status/index.jsx` as well.